strings.txt

```diff
@@ -14107,33 +14107,33 @@
   [house]
     tags = ios, android
     comment = Editable House Number text field (in address block).
-    en = House
-    ru = Дом
-    fr = Immeuble
-    da = Bygning
-    id = Bangunan
-    ko = 건물
-    sv = Byggnad
-    tr = Bina
-    uk = Будинок
-    vi = Tòa nhà
-    hu = Épület
-    de = Gebäude
-    fi = Rakennus
-    cs = Budova
-    it = Edificio
-    nb = Bygning
-    zh-Hant = 樓房
-    zh-Hans = 楼房
-    th = อาคาร
-    ja = 建物
-    ro = Clădire
-    ar = المبني
-    sk = Budova
-    es = Edificio
-    pl = Numer
-    nl = Gebouw
-    pt = Edifício
+    en = House number
+    ru = Номер дома
+    fr = Numéro de la maison
+    da = Et husnummer
+    id = Nomor rumah
+    ko = 집 번호
+    sv = Ett husnummer
+    tr = Bir ev numarası
+    uk = Номер будинку
+    vi = Số nhà
+    hu = Házszám
+    de = Hausnummer
+    fi = Talon numero
+    cs = Číslo domu
+    it = Numero civico
+    nb = Et husnummer
+    zh-Hant = 門牌號碼
+    zh-Hans = 门牌号码
+    th = บ้านเลขที่
+    ja = 番地
+    ro = Număr casă
+    ar = رقم المنزل
+    sk = Číslo domu
+    es = Un número de domicilio
+    pl = Numer domu
+    nl = Huisnummer
+    pt = N.º de porta
```

Comment:
Погодите, номер входа и номер дома это же разные вещи.

> On Apr 7, 2016, at 16:40, therearesomewhocallmetim <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/2817#discussion_r58872020>:
> 
> > +    de = Hausnummer
> > +    fi = Talon numero
> > +    cs = Číslo domu
> > +    it = Numero civico
> > +    nb = Et husnummer
> > +    zh-Hant = 門牌號碼
> > +    zh-Hans = 门牌号码
> > +    th = บ้านเลขที่
> > +    ja = 番地
> > +    ro = Număr casă
> > +    ar = رقم المنزل
> > +    sk = Číslo domu
> > +    es = Un número de domicilio
> > +    pl = Numer domu
> > +    nl = Huisnummer
> > +    pt = N.º de porta
> Quem atribui o nº de porta na minha casa? - Who assigns the port number in my house? <- поиск по фразе nº de porta + гугл-транслейт. Полагаю, перевод в строчках корректен. Алекс, помержишь?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2817/files/1ef069b6adc08e98de7e75e1d61dd373c4bebc3f#r58872020>